portal/src/texts/core/motion.md

```diff
@@ -1,3 +1,15 @@
 ## Bevegelse
 
-I Fremtind ønsker vi å vise at vi er et selskap i bevegelse. Derfor bruker vi animasjon for å hjelpe brukeren med å forstå hva som skjer når de bruker løsningene våre. Vi benytter det sparsomt men godt – det vil si at vi animerer bare når brukeren utfører en handling i en av løsningene våre. Animasjoner skal alltid ha en funksjonell verdi.
+For å sikre at Fremtind fremstår som et selskap i bevegelse med en tydelig retning fremover, er bevegelse definert som et av våre designprinsipper. I ordet ligger det at vi er utforskende, fremtidsrettet og tilpasningsdyktige.
+
+Vi leter stadig etter nye og bedre måter å løse problemer på, og er ikke redde for å utfordre dagens standard. Bruk av animasjon er med på å synliggjøre Fremtind som et selskap i bevegelse. **Animasjoner skal alltid ha en funksjonell verdi**.
+
+### Nøkkelord
+
+-   Alltid i bevegelse
+-   Tilpasningsdyktig
+-   Progressivt
```

Comment:
Bør vi bruke "fremtidsrettet" i stedet for progressivt her, som over?

Reply:
Nå bruker vi jo ikke fremtidsrettet over lenger, men er enig i at det er et bedre ord!